paper-input.html

```diff
@@ -67,6 +67,10 @@
         display: block;
       }
 
+      input {
```

Comment:
I think this actually belongs in `paper-input-container`. There's a bunch of other input styles there too.

Maybe we should do this for `textarea` as well?